test/unit/CMakeLists.txt

```diff
@@ -2,21 +2,32 @@ add_scylla_test(bptree_compaction_test
   KIND UNIT)
 add_scylla_test(bptree_stress_test
   KIND UNIT)
+# TODO multi run support
 add_scylla_test(btree_compaction_test
-  KIND UNIT)
+  KIND UNIT
+  ARGS -c1 -m1G --count=10000 --iter=13
+  ARGS1 -c1 -m1G --count=17 --iter=3)
 add_scylla_test(btree_stress_test
-  KIND UNIT)
+  KIND UNIT
+  ARGS -c1 -m1G --count=4132 --iter=9
+  ARGS1 -c1 -m1G --count=27 --iter=312)
 add_scylla_test(cross_shard_barrier_test
   KIND UNIT)
 add_scylla_test(lsa_sync_eviction_test
-  KIND UNIT)
+  KIND UNIT
+  ARGS -c1 -m100M --count 10 --standard-object-size 3000000
+  ARGS1 -c1 -m100M --count 24000 --standard-object-size 2048
+  ARGS2 -c1 -m1G --count 4000000 --standard-object-size 128)
```

Comment:
I don't understand how this will work. Will the test be run with all 3 different args?

Reply:
yes. it will create three tests.